install_config/topics/configuring_for_vsphere_registry.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * install_config/configuring_for_vsphere.adoc
+
+[id='vsphere-configuring-registry_{context}']
+= Configuring the {product-title} registry for vSphere
+
+
+[source, bash]
+----
+VMFS
+cd /vmfs/volumes/datastore1/
+# VSAN
+cd /vmfs/volumes/vsanDatastore/
+
+# VMFS
+mkdir kubevols # Not needed but good hygiene
+# VSAN
+/usr/lib/vmware/osfs/bin/osfs-mkdir kubevols # Needed
+
+cd kubevols
+
+vmkfstools -c 25G registry.vmdk
+----
```

Comment:
@dav1x Any insight into what the steps above are doing?

Reply:
Sure. If a storage class isnt defined for the vsphere cloud provider, the VMDK has to be manually created. The above step is doing that.